cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -196,6 +196,10 @@ body {
         &-overview {
             display: block;
             border-bottom: 1px solid @gray-40;
+
+            .u-link__disabled {
+                color: @gray !important;
```

Comment:
Is this perhaps wrong at the source? https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/css/enhancements/utilities.less#L192

Reply:
Agreed!

Reply:
+1 I'm planning a new PR that makes these elements a heading or other element and not a disabled link

Reply:
@cfarm You won't want to use a heading elem, I just removed the existing h3 elements because they were in the doc outline.